plugins/hu.bme.mit.massif.simulink.editor/src/hu/bme/mit/massif/simulink/presentation/SimulinkActionBarContributor.java

```diff
@@ -1,3 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2013, Embraer S.A., Budapest University of Technology and Economics
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0 
+ * which accompanies this distribution, and is available at 
+ * http://www.eclipse.org/legal/epl-v10.html 
+ *
+ * Contributors: 
+ *     Abel Hegedus, Akos Horvath - initial API and implementation 
+ *******************************************************************************/
 /**
```

Comment:
Remove empty headers

Reply:
Should I do the same thing for all (model, edit and editor) code?